If "dad.curCharacter" is same as "gf.curCharacter"

rodney528 opened the pull request:
Basically, ya know-how in the tutorial gf is technically the dad? Well, this makes it (I hope) so it can work with anyone other than just gf.

rodney528 committed:
If "dad.curCharacter" is same as "gf.curCharacter"

Basically ya know how in the tutorial gf is technically the dad? Well this makes it (I hope) so it can work with anyone other then just gf.

ImaginationSuperHero528 commented:
> change the "dad.curCharacter.startsWith(gf.curCharacter)" to "dad.curCharacter == gf.curCharacter"
> for every if statement.

That would definitely work better.

rodney528 committed:
Made it if equals now

dad.curCharacter.startsWith(gf.curCharacter) has been replaced with dad.curCharacter == gf.curCharacter

kingyomoma commented on source/PlayState.hx:
```diff
@@ -3698,11 +3698,11 @@ class PlayState extends MusicBeatState
 							dad.alpha = 0.00001;
 							dad = dadMap.get(value2);
-							if(!dad.curCharacter.startsWith(gf.curCharacter)) {
+							if(!dad.curCharacter == gf.curCharacter) {
```
change this to `if (dad.curCharacter != gf.curCharacter)`

ImaginationSuperHero528 replied:
Ok I'll get to it soon.

ImaginationSuperHero528 replied:
At school rn

rodney528 committed:
:P

ImaginationSuperHero528 commented:
:(

rodney528 committed:
Update PlayState.hx

rodney528 committed:
Merge remote-tracking branch 'upstream/main' into if-dad-is-same-as-gf
